cypress/integration/addToMealplan.spec.js

```diff
@@ -0,0 +1,36 @@
+describe('Add and remove recipe to Mealplan', () => {
+  it('Adds recipe to Mealplan ', () => {
+    cy.visit('/recipes');
+
+    cy.get('#recipes li:eq(3) a').click();
+    cy.get('[data-testid="add-to-mealplan"]').click();
+
+    cy.visit('/');
+    cy.get('#meals li').eq(0);
+    cy.get('#meals li:eq(0) input[type=text]').should(
+      'have.value',
+      'Banoffee Pie'
+    );
+  });
+
+  it('Removes recipe from Mealplan', () => {
```

Comment:
Den Test verstehe ich vom lesen alleine nicht ganz. Mich verwirren die Test-IDs, die `add-to-mealplan` heißen, aber der Test besagt, dass du Elemente entfernst 🤔 Kann ich mit dem gleichen Button hinzufügen oder entfernen?
Vielleicht kannst du dem Button eine andere Test-Id geben :)

Reply:
Das stimmt. Hab jetzt noch eine große Umbenennungsaktion gestartet. :) Danke!